R/call_user_fun.R

```diff
@@ -24,6 +26,11 @@
 #'   weight = "hallo"
 #' )))
 call_user_fun <- function(call) {
+  deprecate_inform(
+    when = "1.3.0",
+    what = "call_user_fun()",
+    details = ""
```

Comment:
I feel like we need something for the details especially as we don't have a replacement - @manciniedoardo and @bundfussr wdyt?

Reply:
The soon-to-be-deprecated `call_user_fun`  expects a `call` ( for formal argument `fun`).  If that is ok, I can cut/paste code from `call_user_fun`

Reply:
You could try. In my experience it is hard to indent lines in messages. Thus it may be hard to read. Another option is to link to the code via the release tags, e.g., https://github.com/pharmaverse/admiral/blob/v1.2.0/R/call_user_fun.R.